docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,85 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class, in which each side of children can be specified as relative to sibling view or parent.
+
+**Relative layout example with `Target` and `RelativeOffset`**
+
+![relativeLayoutExamplePreview](./media/relativeLayoutExamplePreview.png)
+
+```xaml
+<View x:Name="Parent">
+    <View.Layout>
+        <RelativeLayout/>
+    </View.Layout>
+
+    <View x:Name="RedRect"   BackgroundColor="1,0,0,1" WidthSpecification="100" HeightSpecification="100"/>
+    <View x:Name="GreenRect"  BackgroundColor="0,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="RedRect" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"
+          RelativeLayout.LeftRelativeOffset="1.0"/>
+    <View x:Name="BlueRect"  BackgroundColor="0,0,1,1" WidthSpecification="100" HeightSpecification="100"
+          RelativeLayout.RightRelativeOffset="1.0" RelativeLayout.LeftRelativeOffset="1.0"
+          RelativeLayout.HorizontalAlignment="End"/>
+    <View x:Name="YellowRect"  BackgroundColor="1,1,0,1" HeightSpecification="100"
+          RelativeLayout.LeftTarget ="Parent" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"/>
+</View>
+```
+
+The `Target` and the `RelativeOffset` properties are used to define a relationship with each other. The above figure shows the relative layout position rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing.
```

Comment:
It would be good to move this paragraph before image at line 5
```suggestion
The `Target` and the `RelativeOffset` properties are used to define a relationship with each other. The following figure shows the relative layout position rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing:
```

So the structure will look like this:

# Relative layout
The `RelativeLayout` is a layout class, in which each side of children can be specified as relative to sibling view or parent. The `Target` and the `RelativeOffset` properties are used to define a relationship with each other. The following figure shows the relative layout position rectangles with `Target` and `RelativeOffset`. The arrows in the preview indicate where the `LeftTarget` and the `RightTarget` is pointing:

 **Relative layout example with `Target` and `RelativeOffset`** 
image

```xaml
<View x:Name="Parent">
    <View.Layout>
        <RelativeLayout/>
    </View.Layout>
    <View x:Name="RedRect"   BackgroundColor="1,0,0,1" WidthSpecification="100" HeightSpecification="100"/>
    <View x:Name="GreenRect"  BackgroundColor="0,1,0,1" HeightSpecification="100"
          RelativeLayout.LeftTarget ="RedRect" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"
          RelativeLayout.LeftRelativeOffset="1.0"/>
    <View x:Name="BlueRect"  BackgroundColor="0,0,1,1" WidthSpecification="100" HeightSpecification="100"
          RelativeLayout.RightRelativeOffset="1.0" RelativeLayout.LeftRelativeOffset="1.0"
          RelativeLayout.HorizontalAlignment="End"/>
    <View x:Name="YellowRect"  BackgroundColor="1,1,0,1" HeightSpecification="100"
          RelativeLayout.LeftTarget ="Parent" RelativeLayout.RightTarget="BlueRect" RelativeLayout.FillHorizontal="true"/>
</View>
```

Reply:
Done :)